src/AppBundle/Search/Repository.php

```diff
@@ -24,17 +29,16 @@ class Repository
      */
     private $repositoryName;
 
-    public function __construct(Search $searchApi, string $repositoryOwner, string $repositoryName)
+    public function __construct(Search $searchApi, GraphQL $graphQL, string $repositoryOwner, string $repositoryName)
```

Comment:
Isn't it a BC to add this parameter and not define a default value ?

Reply:
Yeah, you're right but I don't think it matters as we're talking about PrestonBot here? 🤔

Reply:
Keep always good practices ;)

Reply:
We can say it's a BC and increase the version of the tool :wink: 
But since we are not using prestonbot as a library or a pluggable application it's not a problem, we can break whatever we want :D